systemtests/src/main/java/io/apicurio/registry/systemtest/framework/Utils.java

```diff
@@ -0,0 +1,96 @@
+package io.apicurio.registry.systemtest.framework;
+
+import io.apicurio.registry.systemtest.executor.Exec;
+import io.apicurio.registry.systemtest.platform.Kubernetes;
+import io.apicurio.registry.systemtest.registryinfra.ResourceManager;
+import io.apicurio.registry.systemtest.registryinfra.resources.RouteResourceType;
+import io.apicurio.registry.systemtest.registryinfra.resources.ServiceResourceType;
+import io.apicurio.registry.systemtest.time.TimeoutBudget;
+import io.fabric8.kubernetes.api.model.apps.StatefulSet;
+import io.fabric8.openshift.client.OpenShiftClient;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.slf4j.Logger;
+
+import java.nio.file.Paths;
+import java.time.Duration;
+
+public class Utils {
+    private static final Logger utilsLogger = LoggerUtils.getLogger();
```

Comment:
logger is static value so naming should be something like UTILS_LOGGER. Overall I think it makes very little sense to change the name of the logger. Set loggers as LOGGER everywhere.